### PR TITLE
Completed 2d button which changes mapview between 2d and 3d

### PR DESCRIPTION
**Pull request by hillarytang**
image

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -16,6 +16,7 @@
     <script src="https://cdn-webgl.wrld3d.com/wrldjs/addons/searchbar/latest/searchbar.js"></script>
     <script src="https://cdn-webgl.wrld3d.com/wrldjs/addons/marker_controller/latest/marker_controller.js"></script>
     <script src="https://cdn-webgl.wrld3d.com/wrldjs/addons/indoor_control/latest/indoor_control.js"></script>
+    <script src="https://cdn-webgl.wrld3d.com/wrldjs/dist/latest/wrld.js"></script>
```
Comment (SantiagoOrdonez) [inline public/index.html]: why is this here? We are installing wrld.js using npm and it is being compiled with our code by webpack.

File: src/components/Map.vue [hunk 2]
```diff
@@ -43,6 +46,7 @@
       this.map.indoors.on("collapse", this.onIndoorMapCollapsed);
       this.$refs.scroll.loadScrollbar(this.map);
       this.$refs.search.loadSearchbar(this.map);
+      //this.$refs.tilt.tiltMap(this.map);
```
Comment (SantiagoOrdonez) [inline src/components/Map.vue]: remove commented code

File: src/components/MapViewButton.vue [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<template>
+    <div id="eegeo-tilt-button-container" class="eegeo-tilt-button-container">
+        <button class="eegeo-ripple-button eegeo-tilt-button" v-on:click="toggleView">{{this.mapViewText}}</button>
```
Comment (SantiagoOrdonez) [inline src/components/MapViewButton.vue]: Are these classes from wrld.css?

File: public/index.html (hunk 1 above)
Comment (hillarytang) [reply public/index.html]: For some reason without that line, the button wasn't finding the setTiltDegrees function. But when I added it in it was okay again? I'm not sure

File: src/components/MapViewButton.vue (hunk 3 above)
Comment (hillarytang) [reply src/components/MapViewButton.vue]: They're supposed to be but they also weren't finding them from the stylesheet? So I just copy and pasted the styles into the bottom style tags.

Comment (SantiagoOrdonez) [reply src/components/MapViewButton.vue]: yeah let's investigate this too. Instead of just including it.

File: public/index.html (hunk 1 above)
Comment (SantiagoOrdonez) [reply public/index.html]: mhm we do not need to include this script here because we add wrld as an npm package. Let's investigate what the error is instead. It is good to avoid adding code that we do not quite understand because it will eventually come bite us later.  Ping me on teams with the error.

**Comment by SantiagoOrdonez** [comment]
@hillarytang Your branch has conflicts, see if you can apply what I thought you yesterday and resolve them. Let me know if you  need help :)
